starter/assets/images/empty-state/thinking.svg

```diff
@@ -1,10 +0,0 @@
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 105.104 105.104" fill="#7fbae3">
```

Comment:
If it's the image I think (empty table?) then it should probably be moved to stark-ui (assuming we'll keep it)